crates/testing/src/test_builder.rs

```diff
@@ -70,6 +71,21 @@ pub struct TestDescription {
     pub unreliable_network: Option<Box<dyn NetworkReliability>>,
     /// view sync check task
     pub view_sync_properties: ViewSyncTaskDescription,
+    /// description of builders to run
+    pub builders: Vec1<BuilderDescription>,
+}
+
+#[derive(Clone, Debug)]
+pub enum BuilderChange {
```

Comment:
Can we get a descriptive comment for what this enum is doing and what each of its fields are doing? It's not immediately clear what `Up` and `Down` etc are doing to an actively running builder, and how that affects the test.

Reply:
Good catch! Curiously, clippy doesn't catch this